app/register.js

```diff
@@ -1,24 +1,24 @@
-import React, { useState, useRef } from 'react';
+import React, { useState, useRef } from "react";
```

Comment:
not sure what changes you would be making on this page, please review and remove any changes that are not related to this Item. also, by fixing your formatter, a lot of changes will go away related to quotations